diff --git a/queries/rust/highlights.scm b/queries/rust/highlights.scm
index 82fb966f..d6d3c652 100644
--- a/queries/rust/highlights.scm
+++ b/queries/rust/highlights.scm
@@ -367,18 +367,3 @@
 (macro_invocation "!" @function.macro)
 
 (empty_type "!" @type.builtin)
-
-(macro_invocation
-  macro: (identifier) @exception
-  "!" @exception
-  (#eq? @exception "panic"))
-
-(macro_invocation
-  macro: (identifier) @exception
-  "!" @exception
-  (#contains? @exception "assert"))
-
-(macro_invocation
-  macro: (identifier) @debug
-  "!" @debug
-  (#eq? @debug "dbg"))
